F77.pm

```diff
@@ -712,29 +714,11 @@ sub gcclibs {
 }
 
 # Try and find a program in the users PATH
-
 sub find_in_path {
-   my @names = @_;
-   my @path;
-   if ($^O =~ /mswin32/i) {
-      for(@names) { $_ .= '.exe'}
-      @path = split(";", $ENV{PATH});
-   }
-   else {@path = split(":",$ENV{PATH})}
-   my ($name,$dir);
-   for $name (@names) {
-      return $name if exists $CACHE{$name};
-      for $dir (@path) {
-         if (-x $dir."/$name") {
-            print "Found compiler $name\n";
-            $CACHE{$name}++;
-            return $name;
-         }
-      }
-   }
-   return '' if $^O eq 'VMS';
-   return undef;
-#   die "Unable to find a fortran compiler using names: ".join(" ",@names);
+  my $found = first { which $_ } @_;
+  return ($^O eq 'VMS' ? '' : undef) if !$found;
```

Comment:
Nice simplification! I didn't know about 'first' and 'which' and that these are now standard modules.
Why the VMS specific line?

Reply:
This is just a simplification of the code, which just above the `return undef` above, says:

```
   return '' if $^O eq 'VMS';
```

This is just a shorter way of saying the same thing :-)

Reply:
yes I get the code. I just wondered why VMS was singled out.